sympy/printing/tests/test_octave.py

```diff
@@ -322,6 +374,26 @@ def test_octave_not_supported():
     )
 
 
+def test_octave_expint():
+    assert mcode(expint(1, x)) == "expint(x)"
+    assert mcode(expint(2, x)) == (
+        "% Not supported in Octave:\n"
+        "% expint\n"
+        "expint(2, x)"
+    )
+    assert mcode(expint(y, x)) == (
+        "% Not supported in Octave:\n"
+        "% expint\n"
+        "expint(y, x)"
+    )
+
+
+def test_octave_zeta():
+    assert octave_code(zeta(x)) == 'zeta(x)'
+    n = Symbol('n')
+    assert octave_code(zeta(x, n)) != 'zeta(n, x)'
```

Comment:
In the issue you say "octave_code(zeta(x,n)) should give zeta(n, x)" but here you assert that it is not. Which one is correct?

Reply:
My original issue #14820 was incorrect and I have already updated it.

From the commit msg:
> A previous commit mapped `zeta(x, n)` to `"zeta(n, x)"`: this is incorrect
> because in Matlab's Symbolic Toolbox, this means the nth derivative of the
> Riemann zeta function.  But for us, its the Hurwitz zeta function.
> Instead, just mark the two-arg zeta as unimplemented.